### PR TITLE
Add Wikimedia presence

### PR DESCRIPTION
**Pull request by Hans5958**
*Related: Hans5958/PreMiD-Presences-Personal#1*

## Description

Here's a long overdue of Project Wikimedia. For the sake of completeness, here's a catch-all presence for other Wikimedia domains. These does not include Wikipedia and the others. Compared to other domains, the scope is wider than the other wikis. 

Along with the new presence is the experiment of using the MediaWiki configuration values exposed in `mw.config`. It's my first time of me using `presence.getPageletiable()`, and my first impressions is not quite good, but I get it done nonetheless. If this is merged, then I will also include this update to the other MediaWiki presences.

Notes:
- Should I use Wikimedia or Wikimedia Foundation? Wikimedia refers to the movement, and the Wikmedia Foundation refers to the foundation that's doing the foundation.
- All domains that have been added has been tested so it works properly. Using Regex does not gurantee support.

### Side-by-side

| Image | Link visited |
| ----- | ------------ |
| 
 | https://meta.wikimedia.org/ |

### Variations

| Image | Link visited |
| ----- | ------------ |
|  | https://www.wikimedia.org/ |
|  | https://wikimediafoundation.org/ |
|  | https://wikimediafoundation.org/wikipedia20/ |
|  | https://diff.wikimedia.org |
|  | https://diff.wikimedia.org/2021/10/03/join-us-celebrate-the-eureka-effects-of-the-3rd-edition-of-wiki-loves-folklore/ |
|  | https://techblog.wikimedia.org/ |
|  | https://techblog.wikimedia.org/2021/09/28/how-we-deploy-code/ |
|  | https://lists.wikimedia.org/postorius/lists/ |
|  | https://lists.wikimedia.org/postorius/lists/wikimedia-l.lists.wikimedia.org/ |
|  | https://stats.wikimedia.org/#/all-projects |
|  | https://stats.wikimedia.org/#/all-projects/contributing/edits/normal|bar|2-year|~total|monthly |
|  | https://phabricator.wikimedia.org/ |
|  | https://phabricator.wikimedia.org/T292358 |
|  | https://phab.wmflabs.org/ |
|  | https://xtools.wmflabs.org/ |
|  | https://xtools.wmflabs.org/authorship/en.wikipedia.org/Discord%20%28software%29/ |
|  | https://dumps.wikimedia.org/ |
|  | https://donate.wikimedia.org/ |
|  | https://meta.wikimedia.org/ |
|  | https://meta.wikimedia.org/wiki/Growing_Wikimedia |
|  | https://incubator.wikimedia.org/ |
|  | https://wikitech.wikimedia.org/ |
|  | https://www.mediawiki.org/ |
|  | https://wikimania.wikimedia.org/ |
|  | https://foundation.wikimedia.org/ |
|  | https://wikispore.wmflabs.org/ |
|  | https://outreach.wikimedia.org/ |
|  | https://strategy.wikimedia.org/ |

### CODE REVIEW

File: websites/W/Wikimedia/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "stats.wikimedia.org",
+    "phabricator.wikimedia.org",
+    "phab.wmflabs.org",
+    "dumps.wikimedia.org",
+    "diff.wikimedia.org",
+    "techblog.wikimedia.org",
+    "xtools.wmflabs.org",
+    "donate.wikimedia.org",
+    "wikispore.wmflabs.org"
+  ],
```
Comment (EncryptedDev) [inline websites/W/Wikimedia/dist/metadata.json:45]: Regex?

File: websites/W/Wikimedia/presence.ts [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+          // return details[mwConfig.wgNamespaceNumber] || `Viewing ${detailsShort[namespaceName.split(" talk")[0]] + (namespaceName.endsWith(" talk") ? " talk" : "") || `a/n ${namespaceName}`} page`
+        };
+
+      //
+      // Important note:
+      //
+      // When checking for the current location, avoid using the URL.
+      // The URL is going to be different in other languages.
+      // Use the elements on the page instead.
+      //
```
Comment (EncryptedDev) [inline websites/W/Wikimedia/presence.ts:479]: no comments

File: websites/W/Wikimedia/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "lists.wikimedia.org",
+    "stats.wikimedia.org",
+    "phabricator.wikimedia.org",
+    "phab.wmflabs.org",
+    "dumps.wikimedia.org",
+    "diff.wikimedia.org",
+    "techblog.wikimedia.org",
+    "xtools.wmflabs.org",
+    "donate.wikimedia.org",
+    "wikispore.wmflabs.org"
```
Comment (Bas950) [inline websites/W/Wikimedia/dist/metadata.json:44]: regex?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by Bas950 — DISMISSED** [review]

File: websites/W/Wikimedia/dist/metadata.json (hunk 3 above)
Comment (Hans5958) [reply websites/W/Wikimedia/dist/metadata.json:44]: > All domains that have been added has been tested so it works properly. Using Regex does not gurantee support.

File: websites/W/Wikimedia/dist/metadata.json (hunk 1 above)
Comment (Hans5958) [reply websites/W/Wikimedia/dist/metadata.json:45]: Didn't saw you there. As I said...

> All domains that have been added has been tested so it works properly. Using Regex does not guarantee support.

File: websites/W/Wikimedia/presence.ts (hunk 2 above)
Comment (Hans5958) [reply websites/W/Wikimedia/presence.ts]: Really weird rule you got there... Removed nonetheless.

**Review by EncryptedDev — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]

**Comment by Hans5958** [comment]
@Bas950 it's up to you

**Pull request merged**